tests/core/search/test_search_filter.py

```diff
@@ -26,19 +26,6 @@ def test_parse_successful(self):
                 SearchFilter(root=EqualityTerm('a', 'n', 'a:n')),
                 'single term with extra parentheses'
             ),
-            (
-                'NOT a:1 AND b:2 OR NOT c:3',
```

Comment:
Let's not remove tests we had before this PR. Can change them to have IN if needed